src/main/java/Barney.java

```diff
@@ -12,6 +16,24 @@ public static void welcomeText() {
         System.out.println(LONG_LINE);
     }
 
+    public static void chat() {
+        String input = "";
+        Boolean isChatting = true;
+        while (isChatting) {
+            switch (input) {
+                case "bye":
+                    isChatting = false;
+                    break;
+                default:
+                    System.out.println(">>>");
+                    input = SCANNER.nextLine();
+                    System.out.println("<<<");
+                    System.out.println(input);
+                    System.out.println(LONG_LINE);
+            }
+        }
+    }
+
     public static void endingText() {
```

Comment:
Just a very minor issue here, perhaps the `endingText` method here could be renamed to `endText` to align with the coding standard?